FoodHaus_App/project-swift/Controller/Menu/CheckViewController.swift

```diff
@@ -3,11 +3,18 @@ import Firebase
 
 class CheckViewController: UIViewController {
     
-
+    
+    @IBOutlet weak var totalPrice: UILabel!
+    
+    @IBOutlet weak var tax: UILabel!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        // Do any additional setup after loading the view.
+        if let totalPriceText = totalPrice.text,
```

Comment:
Nice work! However, there's a warning displayed. Please check it out and remove it!